pilot/pkg/config/kube/gateway/testdata/tcp.status.yaml.golden

```diff
@@ -80,3 +80,12 @@ status:
       name: gateway
       namespace: istio-system
 ---
+apiVersion: networking.x-k8s.io/v1alpha1
+kind: TCPRoute
+metadata:
+  creationTimestamp: null
+  namespace: default
+spec: null
+status:
```

Comment:
Is this to test the above change?

I donot think with this config, `buildTCPVirtualService` can run into the clause

Reply:
i modify this file `pilot/pkg/config/kube/gateway/testdata/tcp.yaml` to trigger it